src/commands/game_commands/play.ts

```diff
@@ -17,6 +17,11 @@ import KmqMember from "../../structures/kmq_member";
 const logger = new IPCLogger("play");
 const DEFAULT_LIVES = 10;
 
+export const COMPETITION_MODERATOR_IDS = [
```

Comment:
Let's not hardcode IDs into master

Reply:
I can pass them as environment variables